client/components/TestQueueRow/index.jsx

```diff
@@ -517,39 +517,30 @@ const TestQueueRow = ({
             );
         } else if (runStatus === 'DRAFT' || !runStatus) {
             status = <span className="status-label not-started">Draft</span>;
-        } else if (runStatus === 'IN_REVIEW') {
-            status = (
-                <span className="status-label in-progress">In Review</span>
-            );
-        }
+        } 
 
         return { status, results };
     };
 
     const evaluateNewReportStatus = () => {
-        const { status, conflictsLength } = testPlanReport;
+        const { conflictsLength } = testPlanReport;
         const conflictsCount = conflictsLength;
 
         // If there are no conflicts OR the test has been marked as "final",
```

Comment:
nit: this comment seems to describe when the application could downgrade a Test Plan Report within the Test Queue so this could be removed.